docs/recipes/catalog/github-actions.yaml

```diff
@@ -1,7 +1,7 @@
-recipeName: Deployment from GitHub
+recipeName: Deployment from GitHub Actions
```

Comment:
One more little thing, I would maybe even just called this `Flow Deployment with GitHub Actions` to be more specific about what's happening